docs/gds-supported-platform/tls-certificates.md

```diff
@@ -1,5 +1,10 @@
 # Public TLS Certificates
 
+All apps in GSP are protected by TLS.  There are two options for this:
+using cluster-provided certificates, or providing your own.
```

Comment:
Does the user have to enable the relevant one of these (i.e. TLS isnt enabled until one of these options is enabled)? 

Or is one enabled by default and the user needs to enable the other in order for it to work?